csharp-intermediate-topics/DeepCopyInCSharp/Tests/Test.cs

```diff
@@ -0,0 +1,85 @@
+using DeepCopyInCSharp;
+
+namespace Tests
+{
+    public class Test
+    {
+        private Person _person = new Person
+        {
+            Name = "Steve Doe",
+            Age = 30,
+            Address = new Address
+            {
+                Street = "123 Main St.",
+                City = "Anytown",
+                State = "AB"
+            }
+        };
+
+        [Fact]
+        public void WhenUsingICloneableAllPropertiesCopied()
+        {
+            var copy = (Person)_person.Clone();
```

Comment:
Need a newline here. Same for all tests. Just nicer to split up the Asserts into one block.